prow/jobs/incubator/compass/compass-integration.yaml

```diff
@@ -39,13 +37,13 @@ presubmits: # runs on PRs
     - ^master$
     <<: *vm_job_template
     # following regexp won't start build if only Markdown files were changed
-    # run_if_changed: "^((chart/\\S+|installation/\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))" # TODO: Uncomment when done testing
+    run_if_changed: "^((chart/\\S+|installation/\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))"
```

Comment:
Can we simplify it?

Reply:
Hard to simplify it without negative lookahead but I found some edge cases in that regex and fixed them.